gcloud-java-storage/src/main/java/com/google/gcloud/storage/spi/DefaultStorageRpc.java

```diff
@@ -12,25 +12,9 @@
  * the License.
  */
 
-package com.google.gcloud.spi;
+package com.google.gcloud.storage.spi;
 
 import static com.google.common.base.MoreObjects.firstNonNull;
-import static com.google.gcloud.spi.StorageRpc.Option.DELIMITER;
```

Comment:
We could keep these static imports, right?  I slightly prefer keeping them so that the code down below is a tad more concise.